CHANGELOG.rst

```diff
@@ -14,16 +14,25 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+
+[3.1.0] - 2023-07-21
+~~~~~~~~~~~~~~~~~~~~
+
+Added
++++++
+* Added Django42 support in CI.
+* Removed old versions of celery and drf version from ci.
+
 Removed
 +++++++
-
 * Removed Python 3.5 support.
```

Comment:
Kindly add new entry for the 3.0.0 release in the changelog. And move Python 3.5 support point and celery 5.x to that section as this is already released then but changelog was not updated.

Reply:
done